docs/src/pages/docs/reference/tools/vector-query-tool.mdx

```diff
@@ -122,6 +122,25 @@ The default tool description provides:
 - Instructions for querying the specified vector store
 - Examples of filter syntax when filtering is enabled
 
+## Vector Store Prompts
+
+Each vector store has optimized prompts available in `@mastra/rag`:
+
+```typescript
+// For PostgreSQL with pgvector
+import { PGVECTOR_PROMPT } from '@mastra/rag';
+// For Pinecone
+import { PINECONE_PROMPT } from '@mastra/rag';
+// For Qdrant
+import { QDRANT_PROMPT } from '@mastra/rag';
+```
+
```

Comment:
How do we use these?

Reply:
I would also say specifically: "These are needed when filtering to let the agent know which operators are valid for their vector store"